src/gaTracker/helpers.ts

```diff
@@ -80,6 +80,7 @@ export const ga4EventParams = (): GA4EventParameters => {
 		// littledata_client_id: window.LittledataLayer?.attributes?.['_google-clientID'],
 		sent_from: 'Littledata script',
 		send_to: window.LittledataLayer.measurementId,
+		store_name: window.Shopify?.shop,
```

Comment:
```suggestion
		...(window.Shopify?.shop && { store_name: window.Shopify?.shop }),
```

what if it's undefined?